libcloudforensics/providers/gcp/internal/project.py

```diff
@@ -115,3 +117,17 @@ def log(self) -> log_module.GoogleCloudLog:
     self._log = log_module.GoogleCloudLog(  # type: ignore
         self.project_id)
     return self._log  # type: ignore
+
+  @property
+  def monitoring(self) -> monitoring_module.GoogleCloudMonitoring:
+    """Get a GoogleCloudMonitoring object for the project.
+
+    Returns:
+      GoogleCloudMonitoring: Object that represents Google Monitoring.
+    """
+
+    if self._monitoring:
+      return self._monitoring
+    self._monitoring = monitoring_module.GoogleCloudMonitoring(  # type: ignore
+        self.project_id)
+    return self._monitoring  # type: ignore
```

Comment:
Why are you ignoring the type here? Can't you set the type to `monitoring_module.GoogleCloudMonitoring`? @hacktobeer ran into something similar lately

Reply:
yeah, it's weird. type checking does not recognise the return value of monitoring_module.GoogleCloudMonitoring()....

Reply:
I opened https://github.com/google/cloud-forensics-utils/issues/167 to track this. Feel free to ignore for this PR